client/src/LogOut.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import './App.css';
+
+function LogOut() {
+
+  return (
+    <div className="login-container logout column">
+      <h1>You have been successfully logged out!</h1>
+      <h4>Click to log in</h4>
```

Comment:
If it isn't a title, it should not be `h1` or `h4`. Generally, do not use semantically wrong tags just because of their default styling (larger font size) but style it yourself.

Also it isn't a clickable link so _Click to log in_ label is misleading.